lib/ui/painting.dart

```diff
@@ -2633,6 +2633,7 @@ class ColorFilter {
 
   // The type of SkColorFilter class to create for Skia.
   // These constants must be kept in sync with ColorFilterType in paint.cc.
```

Comment:
This line is no longer true and should just be removed.

Reply:
Done